models/FirstNmfProject.md

```diff
@@ -1,16 +1,19 @@
 # Starting our first project
 
 The example we are using here is a simple and very abstract model of a company. First, let's take a look at the corresponding UML Diagram:
-![Alt Text](images/UmlDiagram.png)
+![Example Uml](images/UmlDiagram.png)
 
 From this class diagram, we use the EMF Editor to create the corresponding Ecore file and the appropriate XMI files, which we then copy into a model directory in our NMF project.
-![Alt Text](images/ModelFolder.png)
+![Folder structure](images/ModelFolder.png)
 
 Now we change to our model directory in the terminal and execute our Ecore2Code command there to generate the corresponding classes for our project. 
 As for the command itself, it copies the ecore file and creates a new file with the extension .nmf (which we define ourselves) and adapts this file so that it works for NMF. Next, the command generates the corresponding C# classes from it.
 ```bash
 Ecore2Code -f -n NMFDemo -o metaModel -m company.nmf company.ecore 
 ```
+For further explanation checkout this link:
```

Comment:
Ich würde immer auch beschreiben, was der Leser unter diesem Link vorfinden wird, also "For further information, find an explanation of the command-line parameters here", wobei der letzte Teil des Satzes dann halt der Link ist